README.md

```diff
@@ -60,6 +60,65 @@ export IAM_STORAGE_FILE_PATH=dist/authN_TempStorageFile
 
 ## Integration Tests
 
+## Startup supporting services
+
+Use https://github.com/mojaloop/platform-shared-tools/tree/main/packages/deployment/docker-compose-infra
+
+
+To startup Kafka, MongoDB, Elasticsearch and Kibana, follow the steps below(in docker-compose-infra/):   
+
+1. Create a directory called `exec` inside the `docker-compose-infra` (this) directory, and go to that directory.
```

Comment:
1. Change `Create a directory called exec in` to `Create a sub-directory called ...`
2. Change `, and go to that direct` to `, and navigate to that direct ...`